wallet/lib/pages/presenting_onboard_page/screens/accept_policy_screen.dart

```diff
@@ -0,0 +1,249 @@
+                                ImageUtil.PYLONS_LOGO,
+                              ),
+                            ),
+                            MyCheckBox(
+                              LocaleKeys.acknowledge_terms_service.tr(),
+                              isSelected: viewModel.isCheckTermServices,
+                              onChange: (value) {
+                                viewModel.toggleCheckTermServices(value!);
+                              },
+                              onLinkTap: () {
+                                /// TO DO
+                                /// Don't have url of term & services
+                              },
+                            ),
+                            SizedBox(
+                              height: 15.0.h,
+                            ),
+                            MyCheckBox(
+                              LocaleKeys.acknowledge_privacy_policy.tr(),
+                              isSelected: viewModel.isCheckPrivacyPolicy,
+                              onChange: (value) {
+                                viewModel.toggleCheckPrivacyPolicy(value!);
+                              },
+                              onLinkTap: () => launchUrlString(kPrivacyPolicyLink),
+                            ),
+                            SizedBox(
+                              height: 25.0.h,
+                            ),
+                            PylonsGetStartedButton(
+                              key: const Key(kAcceptBottomSheetBtnKey),
+                              enabled: viewModel.isCheckTermServices && viewModel.isCheckPrivacyPolicy,
+                              onTap: () async {
+                                viewModel.setUserAcceptPolicies();
+                                await navigatorKey.currentState!.pushNamed(RouteUtil.ROUTE_PURCHASE_VIEW, arguments: widget.nft);
+                              },
+                              text: LocaleKeys.get_started.tr(),
+                              loader: ValueNotifier(false),
+                              fontWeight: FontWeight.normal,
+                              btnHeight: 40,
+                              btnWidth: 260,
+                              btnUnselectBGColor: AppColors.kDarkDividerColor,
+                              fontSize: 14,
+                              textColor: !(viewModel.isCheckTermServices && viewModel.isCheckPrivacyPolicy) ? AppColors.kUserInputTextColor : AppColors.kWhite,
+                            ),
+                          ],
+                        ),
+                      ),
+                    ),
+                  );
+                },
+              );
+            },
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget getTypeWidget(NFT nft) {
+    switch (nft.assetType) {
+      case AssetType.Image:
+        return imageWidget(nft.url);
+      case AssetType.Audio:
+      case AssetType.Video:
+      case AssetType.Pdf:
```

Comment:
I don't think Pdf has a thumbnail url. Are you sure?

Reply:
Yes PDF has thumbnail URL, we have forced user to upload thumbnail too in case of PDF.